scheduler.rst

```diff
@@ -168,22 +175,67 @@ Then, define the trigger date/time using the same syntax as the
 
     RecurringMessage::cron('* * * * *', new Message());
 
+.. tip::
+
+    Check out the `crontab.guru website`_ if you need help to construct/understand
+    cron expressions.
+
 You can also use some special values that represent common cron expressions:
 
-* ``#yearly``, ``#annually`` - Run once a year, midnight, Jan. 1 - ``0 0 1 1 *``
-* ``#monthly`` - Run once a month, midnight, first of month - ``0 0 1 * *``
-* ``#weekly`` - Run once a week, midnight on Sun - ``0 0 * * 0``
-* ``#daily``, ``#midnight`` - Run once a day, midnight - ``0 0 * * *``
-* ``#hourly`` - Run once an hour, first minute - ``0 * * * *``
+* ``@yearly``, ``@annually`` - Run once a year, midnight, Jan. 1 - ``0 0 1 1 *``
+* ``@monthly`` - Run once a month, midnight, first of month - ``0 0 1 * *``
+* ``@weekly`` - Run once a week, midnight on Sun - ``0 0 * * 0``
+* ``@daily``, ``@midnight`` - Run once a day, midnight - ``0 0 * * *``
+* ``@hourly`` - Run once an hour, first minute - ``0 * * * *``
 
 For example::
 
-    RecurringMessage::cron('#daily', new Message());
+    RecurringMessage::cron('@daily', new Message());
+
+Hashed Cron Expression
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you have many trigger scheduled at same time (for example, at midnight, ``0 0 * * *``)
+this could create a very long running schedule right at this time.
+This may cause an issue if a task has a memory leak.
```

Comment:
I don't like this sentence. Maybe we should talk about real use case.
If I remember fabpot talk about email campaign

Reply:
what about writing bottle neck like lot of I/O, memory spike, etc etc?